### ISSUE TITLE
When using `replication-manager-cli` to set maintenance, specify 'on' or 'off' and proxysql state to go OFFLINE_SOFT

### ISSUE THREAD

**Issue by peterpan104** [signal18/replication-manager#688]
It's just like:
```
## set maintenance
replication-manager-cli server --cluster $CLUSTER --id $S_ID --maintenance on

## set online
replication-manager-cli server --cluster $CLUSTER --id $S_ID --maintenance off

```

The current issue is that openstate might be triggered at the same time, causing maintenance to be executed repeatedly, causes the server to remain online.

The following is the openstate script:
```
ERR_LIST=(ERR00042 ERR00087 ERR00040)

## if slave err  -> set slave to maintenance
if [[ ${ERR_LIST[*]} =~ $ERR_NO ]]
then
    echo -e "== START $ERR_NO TIME : $(date +'%Y-%m-%d %H:%M:%S') ==\nOPEN $ERR_NO." >> $LOG 2>&1

    IS_MNT=$(replication-manager-cli topology --cluster $CLUSTER |egrep $S_IP |awk '{print $11}')
    if [[ $IS_MNT != "Y" ]]
    then
        echo "SET SLAVE:$S_IP TO MAINTENANCE." >> $LOG 2>&1
        ## set maintenance
         S_ID=$(replication-manager-cli topology --cluster $CLUSTER |grep $S_IP |awk '{print $1}')
        replication-manager-cli server --cluster $CLUSTER --id $S_ID --maintenance >> $LOG 2>&1
    else
        echo "$CLUSTER:$S_IP IS MAINTENANCE." >> $LOG 2>&1
    fi
```

When ERR_NO is triggered, it checks if `MAINT` = N and then executes the maintenance setting. 

However, when two ERR_NO are triggered simultaneously,
This results in repeated execution of
`replication-manager-cli server --cluster $CLUSTER --id $S_ID --maintenance`

Is it possible to control this by specifying 'on' or 'off'?

**Comment by peterpan104** [comment]
hi @svaroqui 

When I use the web to set up maintenance, I can see that my proxysql will be set to OFFLINE_SOFT.
But using replication-manager-cli to set maintenance does not change anything. 
The previous release didn't have this problem. Is it possible this can be fixed?

![image](https://github.com/signal18/replication-manager/assets/140467680/82831cdb-4638-41c0-bc3d-00c59c888ba3)

<img width="663" alt="image" src="https://github.com/signal18/replication-manager/assets/140467680/fb7b0d67-e460-4745-8f72-180a69bb7fbd">

**Comment by svaroqui** [comment]
What command are you using i've change the command notice the --set="maintenance=on" ? 
maintenance=off|maintenance=switch
```
./replication-manager-cli server --help 
The server command is used to stop , start or put a server in maintenace

Usage:
  replication-manager-cli server [flags]

Flags:
      --action string     stop|start
      --cert string       Public certificate
      --cluster string    Cluster (default is none)
      --get string        processlist|slow-query|digest-statements-pfs|errors|status-delta|innodb-status|variables|meta-data-locks|query-response-time
  -h, --help              help for server
      --host string       Host of replication-manager (default "127.0.0.1")
      --id string         server id
      --insecure          Don't check certificate (default true)
      --password string   Paswword of replication-manager (default "repman")
      --port string       TLS port of  replication-manager (default "10005")
      --set string        maintenance=on|maintenance=off|maintenance=switch|ignored=switch|prefered=switch
```

**Comment by peterpan104** [comment]
Hi @svaroqui 

Server 172.24.44.182 uses web settings, proxysql will change.
Server 172.24.47.188 set using `replication-manager-cli server --cluster $cluster --id $id --set maintenance=on`, proxysql will not change.

In older release,if use `replication-manager-cli server --cluster $cluster --id $id --maintenance` , proxysql will be changed.

**Comment by peterpan104** [comment]
hi @svaroqui 
Is there any chance this problem can be solved? Or should I open a new issue?

**Comment by svaroqui** [comment]
This was solved 
./replication-manager-cli server --cluster=bench  --id=db5882483430095052171 --set maintenance=on

![Capture d’écran 2024-09-13 à 18 35 57](https://github.com/user-attachments/assets/a04552f3-8878-4e1b-96cb-d4ef319b32f3)

**Comment by svaroqui** [comment]
![Capture d’écran 2024-09-13 à 18 39 13](https://github.com/user-attachments/assets/5c32d717-501c-401e-bbe2-1d0521bd17a0)

**Comment by peterpan104** [comment]
hi @svaroqui 

I see you changed the title, I think you have found the problem.
Thanks for your reply.

**Comment by svaroqui** [comment]
Hi, Indeed we introduce a doctrine to not touch the proxy when bootstrap is off so in previous release  a maintenance state was pushdown to proxies in any case for monitoring to align the replication-manage state with the state seen by the proxy. Keep in mind that your position is dangerous as it's a one shoot gun if a proxy is stopped and restarted on different topology the it will be ProxySQL monitor to realign

**Comment by svaroqui** [comment]
Please reopen if not fix  in v2.3.47
